service/src/main/java/org/mskcc/cmo/metadb/service/impl/SampleServiceImpl.java

```diff
@@ -42,6 +42,7 @@ public MetaDbSample saveSampleMetadata(MetaDbSample
             if (patient != null) {
                 updatedMetaDbSample.setPatient(patient);
             }
+            patientRepository.save(updatedMetaDbSample.getPatient());
```

Comment:
does this not return a `MetaDbPatient` object which would have a UUID assigned to it?

If not then that's probably what we want. Also the save should only happen if the patient was not found in the db so it should be placed in an `else` block for this `if` statement